README.md

```diff
@@ -123,13 +123,17 @@ Phylum is written in Rust, so you'll need a recent Rust installation to build it
    git clone https://github.com/phylum-dev/cli
    ```
 
-2. Run build and install scripts in cli/lib
+2. Build the project
 
-   ```sh
-   cd cli/lib
-   bash build.sh
-   bash install.sh
-   ```
+```sh
+cargo build
+```
+
+3. You can use the executable directly as `./target/debug/phylum` or install it like so:
```

Comment:
It's true that the executable can be run directly, but without a valid configuration file much of the functionality is unavailable. Here is an example output...but most of the commands will say the same thing:
```
➜  cli git:(kyle/m1-build) ./target/debug/phylum version
[2022-03-16T22:45:36Z ERROR phylum] Execution failed: Failed to read configuration at `/Users/coggins/.phylum/settings.yaml`: No such file or directory (os error 2)
❗ Error: Execution failed caused by: Failed to read configuration at `/Users/coggins/.phylum/settings.yaml`: No such file or directory (os error 2)
```
Recommend adding language about adding the config file, building the `dist` and installing from that artifact, or otherwise make it clear that full functionality is not possible without the config file.

Reply:
Also...not really part of the ticket this review is meant to cover, but the full local build on Apple M1 systems still does not work for the `x86_64-unknown-linux-musl` target build:
```
➜  cli git:(kyle/m1-build) cargo xtask dist
    Finished dev [unoptimized + debuginfo] target(s) in 0.21s
     Running `target/debug/xtask dist`
23:01:04 [INFO] Building Linux Docker image
[+] Building 54.5s (10/10) FINISHED
 => [internal] load build definition from Dockerfile.linux                                                                       0.0s
 => => transferring dockerfile: 43B                                                                                              0.0s
 => [internal] load .dockerignore                                                                                                0.0s
 => => transferring context: 34B                                                                                                 0.0s
 => [internal] load metadata for docker.io/library/rust:1.58.1-slim-buster                                                       1.8s
 => [1/6] FROM docker.io/library/rust:1.58.1-slim-buster@sha256:39c58aeac4dd24844a837d252fed010034094953ced80c37b82061edd0606d8  0.0s
 => [internal] load build context                                                                                                0.2s
 => => transferring context: 394.17kB                                                                                            0.1s
 => CACHED [2/6] RUN apt-get update && apt-get install -y musl-tools                                                             0.0s
 => CACHED [3/6] RUN rustup target add x86_64-unknown-linux-musl                                                                 0.0s
 => CACHED [4/6] WORKDIR /build/cli                                                                                              0.0s
 => [5/6] COPY . .                                                                                                               0.4s
 => ERROR [6/6] RUN cargo build --release --target x86_64-unknown-linux-musl                                                    51.9s
------
 > [6/6] RUN cargo build --release --target x86_64-unknown-linux-musl:
#10 1.251     Updating crates.io index
#10 31.71     Updating git repository `https://github.com/phylum-dev/phylum-types`
#10 32.42  Downloading crates ...
#10 33.16   Downloaded block-buffer v0.9.0
---TRIMMED---
#10 50.76    Compiling tracing v0.1.31
#10 51.08 error: failed to run custom build command for `ring v0.16.20`
#10 51.08
#10 51.08 Caused by:
#10 51.08   process didn't exit successfully: `/build/cli/target/release/build/ring-fa34e1f30f264c44/build-script-build` (exit status: 101)
#10 51.08   --- stdout
#10 51.08   OPT_LEVEL = Some("3")
#10 51.08   TARGET = Some("x86_64-unknown-linux-musl")
#10 51.08   HOST = Some("aarch64-unknown-linux-gnu")
#10 51.08   CC_x86_64-unknown-linux-musl = None
#10 51.08   CC_x86_64_unknown_linux_musl = None
#10 51.08   TARGET_CC = None
#10 51.08   CC = None
#10 51.08   CROSS_COMPILE = None
#10 51.08   CFLAGS_x86_64-unknown-linux-musl = None
#10 51.08   CFLAGS_x86_64_unknown_linux_musl = None
#10 51.08   TARGET_CFLAGS = None
#10 51.08   CFLAGS = None
#10 51.08   CRATE_CC_NO_DEFAULTS = None
#10 51.08   DEBUG = Some("false")
#10 51.08   CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
#10 51.08
#10 51.08   --- stderr
#10 51.08   running "musl-gcc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "include" "-Wall" "-Wextra" "-pedantic" "-pedantic-errors" "-Wall" "-Wextra" "-Wcast-align" "-Wcast-qual" "-Wconversion" "-Wenum-compare" "-Wfloat-equal" "-Wformat=2" "-Winline" "-Winvalid-pch" "-Wmissing-field-initializers" "-Wmissing-include-dirs" "-Wredundant-decls" "-Wshadow" "-Wsign-compare" "-Wsign-conversion" "-Wundef" "-Wuninitialized" "-Wwrite-strings" "-fno-strict-aliasing" "-fvisibility=hidden" "-fstack-protector" "-g3" "-U_FORTIFY_SOURCE" "-DNDEBUG" "-c" "-o/build/cli/target/x86_64-unknown-linux-musl/release/build/ring-bbe6bc9587b4af16/out/aesni-x86_64-elf.o" "/usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.16.20/pregenerated/aesni-x86_64-elf.S"
#10 51.08   cc1: error: unrecognized command line option '-m64'
#10 51.08   thread 'main' panicked at 'execution failed', /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.16.20/build.rs:656:9
#10 51.08   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
#10 51.08 warning: build failed, waiting for other jobs to finish...
#10 51.80 error: build failed
------
executor failed running [/bin/sh -c cargo build --release --target x86_64-unknown-linux-musl]: exit code: 101
Error: couldn't build linux docker image
```
Just mentioning this one in case there are ready ideas. Feel free to ignore as out of scope.

Reply:
See newly created issues #192 and #193